templates/app-autoscaler-deployment.yml

```diff
@@ -19,9 +19,9 @@ releases:
   sha1: "0024f2d9f8bb3c624a162db4c3a5919388c6d800"
   version: "0.237.0"
 - name: loggregator-agent
-  url: https://bosh.io/d/github.com/cloudfoundry/loggregator-agent-release?v=6.4.1
-  version: 6.4.1
-  sha1: 02ec285cce8fef717ab7baee79a5ed9210a7391c
```

Comment:
We need to think if we want to double-maintain the list of used releases in both bosh templates.

I think I would be in favor of removing the version information from this template as well (and [like in the other template only mentioning the releases](https://github.com/cloudfoundry/app-autoscaler-release/blob/main/templates/app-autoscaler.yml#L4-L16)) and requiring the use of `operations/add-releases.yml` here as well.

Reply:
It seems to have worked!